cube/cube.cpp

```diff
@@ -3394,35 +3733,84 @@ int main(int argc, char **argv) {
 
     demo.init(argc, argv);
 
+    switch (demo.wsi_platform) {
+        default:
+        case (WsiPlatform::auto_):
+            fprintf(stderr,
+                    "WSI platform should have already been set, indicating a bug. Please set a WSI platform manually with "
+                    "--wsi");
```

Comment:
This error and the other similar ones can use a line break at the end so it doesn't mix with the next prompt (also on cpp).